packages/feedback/src/core/createMainStyles.ts

```diff
@@ -21,25 +30,16 @@ export function createMainStyles({ colorScheme, themeDark, themeLight }: Feedbac
   const style = DOCUMENT.createElement('style');
   style.textContent = `
 :host {
-  --z-index: ${themeLight.zIndex};
-  --font-family: ${themeLight.fontFamily};
-  --font-size: ${themeLight.fontSize};
-
-  font-family: var(--font-family);
-  font-size: var(--font-size);
+  --font-family: system-ui, 'Helvetica Neue', Arial, sans-serif;
+  --font-size: 14px;
+  --z-index: 100000;
```

Comment:
Are we removing this as available customization options from constructor?

Reply:
ya, it wasn't on docs.sentry.io, and font stuff wasn't possible to override. So i pulled it out, but it's easy to add back. 

I think i'd like a general-theme section in there that's neither dark or light mode. In my testing i screwed up a couple times by setting something like `themeLight: { foreground: 'red' }` then rendering in dark mode i was confused why i couldn't see my color change. It wouldn't be great to have to specify font in both places, or only in one place and have it copied around imo.

So i was thinking, shorter term at least, we lean into css variables more in the docs, so that's the first stop for people. And that's where everything is available. https://github.com/getsentry/sentry-docs/pull/9961/files